packages/types/src/chain-adapters/index.ts

```diff
@@ -1,17 +1,20 @@
-import { BTCSignTx, ETHSignTx, HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { BTCSignTx, CosmosSignTx, ETHSignTx, HDWallet } from '@shapeshiftoss/hdwallet-core'
 
 import { BIP44Params, ChainTypes, NetworkTypes, SwapperType, UtxoAccountType } from '../base'
 import { ChainAndSwapperSpecific, ChainSpecific } from '../utility'
 import * as bitcoin from './bitcoin'
+import * as cosmos from './cosmos'
```

Comment:
is this path right? these are under `cosmossdk` correct? btw, empty index.ts file in `cosmossdk`. Can either remove and require import at `cosmossdk/{chain}` level or export all chains from `cosmossdk` index file